WIP: KCSEU23 Initial Stub Out (#365)
kubernetes/contributor-site: content/en/events/2023/kcseu/social.md

---
title: Contributor Social
type: docs
weight: 5
draft: true
---

> # Contributor Social at La Casa Del Mar
> 
> <img align="right" src="/events/2023/kcseu/TODO.jpg" width="30%">
> 
> From 6:30 to 8:30pm (and beyond), Kubernetes contributors will be schmoozing,
> eating, playing boardgames, and competing in a pub quiz at [La Casa Del Mar](https://lacasadelamar.com/espacios-patacona/), 
> a cafe and bar by the beach in Valencia.
> 
> The Social is open only to registered attendees of the Kubernetes Contributor
> Summit and possibly their guests (see below).  You must be [registered](https://cvent.me/384mb9)
> before attending.
> 
> ## Location And Transportation
> 
> The Contributor Social will be held at the
> <a href="https://lacasadelamar.com/espacios-patacona/" rel="noopener noreferrer" target="_blank">La Casa de la Mar</a>
> several kilometers from the Feria.
> 
> As such, a bus will leave from The Fira (conference venue) at 6pm on May 16th
> to take contributors to the Social.  The bus will return after 8:30pm.
> 
> Some contributors will leave from their hotels; if you want to share a cab from
> your hotel or another location, 
> [sign up on the cab-sharing sheet](https://bit.ly/cseu22carpools).
> 
> You are also welcome to get there on your own.  There aren't any good public 
> transportation links to the venue.

Reviewer: ```suggestion
# Contributor Social

We are still reviewing locations, but should have more info soon! You can expect good food, fun activities and of course, the best company!
```

> ## Activities
> 
> Games!  Prizes!
> 
> Lachlan Evenson will lead a live pub quiz of Kubernetes trivia.  Play alone'
> or in teams; the winning team/player will get prizes.
> 
> Contributors are urged to bring boardgames with them to the Social.  The venue
> has many small tables perfect for playing games with a few other contributors.
> 
> For first-time Contributor Summit attendees, we'll also have Bingo Cards
> intended to help you break the ice with more experienced contributors. The
> first folks to complete vertical row, horizontal row, and blackout on their 
> bingo cards will win prizes.
> 
> ## Food & Drink
> 
> La Casa Del Mar serves American-Spanish hybrid pub food, and we'll have substantial
> snacks available for all contributors, as well as alcoholic and non-alcoholic 
> drinks.

Reviewer: ```suggestion
```

## Bringing A Guest

> Contributors will be able to bring a single family member or 
> partner to the Social.  This does not include coworkers or casual friends;
> our intention is to support domestic partners and close family of contributors
> by including them.  
> 
> Guests must be fully vaccinated and adhere to all other KubeCon health & 
> safety requirements.
> 
> If you would like to bring a guest, [please fill out the request form](https://www.surveymonkey.com/r/kccseu22social).
> 
> If you have questions about guests,
> then [contact us](mailto:community@cncf.io).

Reviewer: ```suggestion
Contributors will be able to bring a single family member or  partner to the
Social. This does not include coworkers or casual friends; our intention is
to support domestic partners and close family of contributors by including
them.

**NOTE:** Guests must adhere to all KubeCon health & safety requirements.

To bring a guest, please email community@kubernetes.io with their information.
```

From NA guidelines.
